ray-curator/ray_curator/stages/io/writer/parquet.py

```diff
@@ -0,0 +1,34 @@
+from dataclasses import dataclass, field
+from typing import Any
+
+from ray_curator.tasks import DocumentBatch
+
+from .dataframe import BaseWriter
+
+
+@dataclass
+class ParquetWriter(BaseWriter):
+    """Writer that writes a DocumentBatch to a Parquet file using pandas."""
+
+    # Additional kwargs for pandas.DataFrame.to_parquet
+    parquet_kwargs: dict[str, Any] = field(default_factory=dict)
+    file_extension: str = "parquet"
+
+    @property
+    def name(self) -> str:
+        return "parquet_writer"
+
+    def write_data(self, task: DocumentBatch, file_path: str) -> None:
+        """Write data to Parquet file using pandas DataFrame.to_parquet."""
+        df = task.to_pandas()  # Convert to pandas DataFrame if needed
+
```

Comment:
Do you prefer this over the direct pa.Table -> file codepath? 
Maybe doesn't need to be done now but benchmarked and addressed in a followup?